payments/admin.py

```diff
@@ -1,6 +1,29 @@
+import json
+
 from django.contrib import admin
-from payments.models import StripeAccount, StripeEvent
+from .models import StripeAccount, StripeEvent
+
+
+class StripeAccountAdmin(admin.ModelAdmin):
+
+    readonly_fields = (
+        'user', 'account_id', 'secret_key', 'public_key',
```

Comment:
Setting the model to editable=False and removing it from the readyonly_fields list hides the secret_key from admin.